render/fragment.js

```diff
@@ -3,8 +3,8 @@
 var Vnode = require("../render/vnode")
 var hyperscriptVnode = require("./hyperscriptVnode")
 
-module.exports = function() {
-	var vnode = hyperscriptVnode.apply(0, arguments)
+module.exports = function(attrs, ...children) {
+	var vnode = hyperscriptVnode(attrs, children)
```

Comment:
Is this truly faster than just spreading the arguments and extracting them in `hyperscriptVnode` via a rest parameter? I'm deeply skeptical, given they all lie on the stack.